contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -943,36 +952,37 @@ contract DualAggregator is OCR2Abstract, Ownable2StepMsgSender, AggregatorV2V3In
   // │                   v3 AggregatorInterface                     │
   // ================================================================
 
+  error RoundNotFound();
+
   uint8 private immutable i_decimals;
 
-  /// @return answers are stored in fixed-point format, with this many digits of precision
+  /// @notice aggregator contract version.
+  uint256 public constant VERSION = 6;
+
+  // Human readable description.
+  string internal s_description;
+
+  /// @return answers are stored in fixed-point format, with this many digits of precision.
   function decimals() public view virtual override returns (uint8) {
     return i_decimals;
   }
 
-  /// @notice aggregator contract version
-  uint256 public constant VERSION = 6;
-
   function version() public view virtual override returns (uint256) {
```

Comment:
Use the inheritdoc tags here?

Reply:
No, because we don't have any natspec in the interface contract.
I decided to only inheritdoc from real contracts, not interfaces since we don't have any comment.